### PR TITLE
G2P release candidate

### PR DESCRIPTION
**Pull request by bwalsh**
This release is coupled with the ga4gh.schemas/g2p and ga4gh.compliance/g2p repos.
It contains:
* protocol buffer conversion
* refactored API
* sequenceAnnotations featureset implementation for feature search

@david4096 @jeromekelleher @kozbo  @diekhans :  can you take a look?  If possible I'd like to merge as soon as practical.   Thanks.

### CODE REVIEW

File: ga4gh/datamodel/__init__.py [hunk 1]
```diff
@@ -324,25 +324,6 @@ class PhenotypeAssociationCompoundId(CompoundId):
         ('phenotypeAssociationId', 2)]
 
 
-class PhenotypeAssociationSetCompoundId(CompoundId):
```
Comment (david4096) [inline ga4gh/datamodel/__init__.py]: There are compound ID classes for the association and association set. Should there be entries for your other identifiers?

**Comment by david4096** [comment]
Seems like some minor changes to the feature branch +1 from me.

**Comment by jeromekelleher** [comment]
These look like minor changes to me too, +1.

Is the full PR for merging in G2P into master nearly ready @bwalsh?

**Comment by bwalsh** [comment]
@jeromekelleher : yes with this PR, we are ready to make PR to master on server/schema/compliance.  Thanks for the +1

File: ga4gh/datamodel/__init__.py (hunk 1 above)
Comment (bwalsh) [inline ga4gh/datamodel/__init__.py]: @david4096: thanks for the +1.  Regarding additional entries for other identifiers, I considered it for Phenotype.  After looking at the use cases, I didn't see the need.  The most likely future use that might require it is GET ../genotype/[id]. I'd recommend deferring that until the need materializes.

**Comment by dcolligan** [comment]
Is there a reason we haven't merged this into the g2p branch yet?

**Comment by bwalsh** [comment]
Not that I'm aware.

-Brian Walsh

**Pull request merged**